propagation.go

```diff
@@ -115,9 +115,9 @@ func (p *textMapPropagator) Inject(
 	// Do not encode the string with trace context to avoid accidental double-encoding
 	// if people are using opentracing < 0.10.0. Our colon-separated representation
 	// of the trace context is already safe for HTTP headers.
-	textMapWriter.Set(TracerStateHeaderName, sc.String())
+	textMapWriter.Set(p.tracer.headerKeys.TracerStateHeaderName, sc.String())
```

Comment:
@black-adder you are correct that `p.tracer.headerKeys.etc` is the only place where the tracer is being used. However, this code was there before this PR. Was there a reason for this to your knowledge? If not, I'll go ahead and change that.